### PR TITLE
Feature/secret

### PR DESCRIPTION
**Pull request by shibuiwilliam**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description
- Added access client to secret manager to retrieve mysql username and password from AWS secret manager.

## Related Tickets & Documents

None

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

None

### CODE REVIEW

**Comment by shibuiwilliam** [comment]
@vios-fish 
Please review

File: automan/automan_website/settings.py [hunk 1]
```diff
@@ -112,12 +113,15 @@
...
 # MySQL
+mysql_access_dict = secret.get_mysql_username_password()
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': os.environ.get("MYSQL_DB_NAME"),
-        'USER': os.environ.get("MYSQL_USER"),
-        'PASSWORD': os.environ.get("MYSQL_PASSWORD"),
+        # 'USER': os.environ.get("MYSQL_USER"),
+        # 'PASSWORD': os.environ.get("MYSQL_PASSWORD"),
```
Comment (vios-fish) [inline automan/automan_website/settings.py:122]: It's okay to have this comment out turned off.

**Review by vios-fish — CHANGES_REQUESTED** [review]
Please check my comment.

File: automan/automan_website/settings.py (hunk 1 above)
Comment (shibuiwilliam) [reply automan/automan_website/settings.py]: thanks. got it.

Comment (vios-fish) [reply automan/automan_website/settings.py]: so quickly! thanks!!

**Review by vios-fish — APPROVED** [review]
LGTM

**Pull request merged**